src/cards/Colors.java

```diff
@@ -0,0 +1,68 @@
+package cards;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.TreeMap;
+
+public enum Colors {
+
+    TREFL("żołądź", "clubs"),
+    KARO("dzwonek", "diamonds"),
+    KIER("serce", "hearts"),
+    PIK("wino", "spades");
+
+    private final String polishDescription;
+    private final String englishDescription;
+    private final String[] polishFigures;
+    private final String[] englishFigures;
+//    private final String[] numberedCards;
+    public static final String POLISH = "PL";
+    public static final String ENGLISH = "EN";
+
+
+    Colors(String polishDescription, String englishDescription) {
+        this.polishDescription = polishDescription;
+        this.englishDescription = englishDescription;
+        this.polishFigures = new String[]{"dwójka","trójka","czwórka","piątka","szóstka","siódemka","ósemka","dziewiątka","dziesiątka","walet","dama","król","as"};
```

Comment:
zamiast trzymac figury w tablicy lepiej stworzyc osobnego enuma na figury

Reply:
poprawione, dodalam nowa wersje tego zadania - package cardsVersion2

Reply:
nie wiem teraz jak polaczyc to wszystko: 
-czy klasa Card powinna zawierać pola typu enum czy pola typu String?
-czy utworzyc obiekt karta i tłumaczyć od razu jej elementy (włożyć do obiektu pole color i figure typu String) a potem włożyć karty do kolekcji? 
-czy może obiekt karta powinien przechowywać pola typu enum i dopiero potem nalezy te pola przetlumaczyc i wlozyc do kolekcji?
-czy tak jak robiłam poprzednio - od razu wkładać tłumaczone elementy do mapy?

Reply:
zalecane przez nas rozwiazanie wygląda tak:
* jeden enum dla kolory karty
* jeden enum dla wartosci karty
* kazdy ztych enumow posiada 2 pola:  z tlumaczeniem polskim i angielskim
* klasa karta ktora reprezentuje jedna karte bedzie miec jendo pole dla koloru i jedno dla wartosci : typy to beda stworzone przez Ciebie enymu

Reply:
poprawione